Source/Core/CorridorGeometry.js

```diff
@@ -805,7 +805,7 @@ define([
         var cleanPositions = PolylinePipeline.removeDuplicates(positions);
 
         if (cleanPositions.length < 2) {
```

Comment:
No need.  Height just determines the offset between the corridor and the ellipsoid surface, so any number makes sense.  Having a negative or zero width, however, is no good.